src/signalr-service.ts

```diff
@@ -0,0 +1,53 @@
+import { Subject } from 'rxjs';
+import { HubConnection, HubConnectionBuilder, LogLevel, HubConnectionState } from '@aspnet/signalr';
+import { AzureWebCommandService } from './azure-web-command-service';
+import { Guid } from 'guid-typescript';
+import { SignalRResponse } from './signal-r-response';
+
+export class SignalRService {
+    private signalrMessage: Subject<any> = new Subject<any>();
+    private hubConnection: HubConnection;
+
+    /**
+     * Create a new instance of SignalRService
+     * @param signalrUrl The azure signalR endpoint.
+     * @param enableLogging Enable logging with level "Debug".
+     */
+    constructor(signalrUrl: string, enableLogging?: boolean) {
+        const connection = new HubConnectionBuilder()
+        .withUrl(signalrUrl);
```

Comment:
Indenting and stuff like these could be resolved by using a linter. We might use something like NgLint for this.

Reply:
The tslint package is available + the npm lint command. Maybe this command can run in the PR and must be successfull before PR can be merged